opensrp-anc/build.gradle

```diff
@@ -21,6 +21,11 @@ android {
     compileSdkVersion androidCompileSdkVersion
     buildToolsVersion androidBuildToolsVersion
 
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
This will mean that everyone else has to update their apps. This does not usually play well for libraries. Mapbox did that and they had to revert back because it caused a lot of issues https://github.com/mapbox/mapbox-gl-native/wiki/Android-6.x-to-7.x-migration-guide#java-7

Reply:
What issues are this and how they affect ANC. 
Supporting java 8 has a lot of advantages that OpenSRP could benefit from. https://code.tutsplus.com/tutorials/java-8-for-android-cleaner-code-with-lambda-expressions--cms-29661